### DISCUSSION TITLE
Deploy > Kubernetes > Content

### DISCUSSION THREAD

**Discussion by sandeep-r1** [Sunbird-Ed/Community#518] [Bugs]
Hi Team,
Content service deployment is failing with below error. Kindly need your help to resolve this issue.
```
TASK [sunbird-deploy : helm install] *******************************************
fatal: [localhost]: FAILED! => {
    "changed": true,
    "cmd": "helm upgrade --install --atomic --timeout 10m content /var/lib/jenkins/workspace/Deploy/alex/dev/Kubernetes/Content/kubernetes/helm_charts/core/content -n alexdev",
    "delta": "0:10:00.571330",
    "end": "2023-06-07 11:06:35.854049",
    "rc": 1,
    "start": "2023-06-07 10:56:35.282719"
}

STDOUT:

Release "content" does not exist. Installing it now.


STDERR:

Error: release content failed, and has been uninstalled due to atomic being set: timed out waiting for the condition


MSG:

non-zero return code
```

**Comment by tsprasath** [comment]
@sandeep-r1 Please share the logs

**Reply by srikanth-6520** [reply]
@tsprasath we have done few changes in configmap of  content service over the cluster level , then job got success.

**Comment by gohilamariappan** [comment]
Hi @tsprasath @santhosh-tg 

We are also facing the same issue , while deploying the content service. pod is crashing and below one is the error.
Please suggest.
################3
error: a container name must be specified for pod content, choose one of: [content envoy opa] or one of the init containers: [proxy-init]
###########

Regards,

Gohila

**Reply by chitranshu-keshav** [reply]
@srikanth-6520 @tsprasath Can you please provide the solution implemented so that others can also replicate.

**Reply by srikanth-6520** [reply]
@chitranshu-keshav , i have faced different issue during the deployment , "write base path and read base path"  , double quotations was missed , so we temporarily updated the missed double quotations for those over the cluster level.

**Reply by Krishnaj20** [reply]
thanks @srikanth-6520 - appreciate your contribution.

@chitranshu-keshav @gandham-santhosh - does this need correction to the microsite or the script? We should get it corrected there as well.

**Reply by chitranshu-keshav** [reply]
@tsprasath can you please get back on the issue @gohilamariappan has reported
@gandham-santhosh can you please get the needful done incase the documentation has to be updated

**Reply by tsprasath** [reply]
@srikanth-6520 Please update the below values in the core-->common.yaml

```
cloudstorage_base_path: "<S3 endpoints>"
valid_cloudstorage_base_urls: '["<S3 endpoints>"]'
```
Eg: for GCP

```
cloudstorage_base_path: "https://storage.googleapis.com"
valid_cloudstorage_base_urls: '["https://storage.googleapis.com"]'
```
Pleas refer private repo template
https://github.com/project-sunbird/sunbird-devops/blob/release-5.1.0/private_repo/ansible/inventory/dev/Core/common.yml#L239-L242

**Reply by srikanth-6520** [reply]
@tsprasath i have added above variables , now the content service is working fine without making the manual changes over the cluster.

**Comment by sandeep-r1** [comment]
Answered

**Reply by gandham-santhosh** [reply]
@Krishnaj20  @chitranshu-keshav  this is already updated in the documentation
